.github/actions/kind-create/action.yaml

```diff
@@ -38,10 +42,12 @@ runs:
       kubectl cluster-info
       kubectl get pods -n kube-system
       echo "current-context:" $(kubectl config current-context)
-      echo "environment-kubeconfig:" ${KUBECONFIG}
-      mkdir tests/e2e/platforms/kind/${{ inputs.kind_k8s_version }}
-      kubeconfig_path=$(pwd)/tests/e2e/platforms/kind/${{ inputs.kind_k8s_version }}/kind.kconf
+      kubeconfig_dir_path=$(pwd)/${{ inputs.kubeconfig_dir_path }}
+      mkdir -p $kubeconfig_dir_path
+      kubeconfig_filename=${{ inputs.kind_cluster_name }}-${{ inputs.kind_k8s_version }}.yaml
```

Comment:
Optional: We only use this variable on the next line, I don't think we really need it, just merge it's value into the next line?

Reply:
It is used here also: https://github.com/banzaicloud/koperator/blob/b678b1756446e619a22cd9024b8298a78c3ae292/.github/actions/kind-create/action.yaml#L48

Reply:
Fair, it's fine then by me.